scene/2d/tile_map.cpp

```diff
@@ -229,8 +229,10 @@ void TileMap::queue_internal_update() {
 	if (pending_update) {
 		return;
 	}
-	pending_update = true;
-	callable_mp(this, &TileMap::_internal_update).call_deferred();
+	if (is_inside_tree()) {
```

Comment:
Could move this to `if (pending_update) {` as `if (pending_update || !is_inside_tree()) {`